packages/node/package-lock.json

```diff
@@ -6358,8 +6368,6 @@
       "integrity": "sha512-D9cPgkvLlV3t3IzL0D0YLvGA9Ahk4PcvVwUbN0dSGr1aP0Nrt4AEnTUbuGvquEC0mA64Gqt1fzirlRs5ibXx8g==",
       "dev": true
     },
-<<<<<<< HEAD
```

Comment:
Not sure how this got through nor why npm wasn't complaining about it when running `npm ci`.